metric-anomaly-task-manager/build.gradle.kts

```diff
@@ -21,8 +21,11 @@ dependencies {
     implementation("org.hypertrace.core.documentstore:document-store:0.5.7")
     implementation("org.hypertrace.config.service:config-service-api:0.1.12")
     implementation("org.hypertrace.config.service:alerting-config-service-api:0.1.12")
+    implementation("org.hypertrace.config.service:notification-rule-config-service-api:0.1.12")
     implementation("org.hypertrace.core.grpcutils:grpc-context-utils:0.5.2")
     implementation("org.hypertrace.core.grpcutils:grpc-client-utils:0.5.2")
+    implementation("org.apache.commons:commons-lang3:3.10")
+    implementation("io.grpc:grpc-netty:1.40.0")
```

Comment:
Do we have to include this explicitly? Isn't it coming for any framework library?

Reply:
No, this has to be explicitly added, the integration test fails otherwise

Reply:
In that case, can it be `integrationTestImplementation` or 
 `testImplementation` ?